manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -70,6 +70,10 @@ type ModuleArguments struct {
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
 
+	// Application selector retrieved from FybrikApplication
```

Comment:
can you document? e.g., Application selector from the FybrikApplication that is a reference to the user workload, and is used to identify the running workload.

Reply:
Done